pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -522,7 +522,8 @@ var _ = Describe("ProjectService", func() {
 					}
 				})
 
-				expectedMaxUnavailable := intstr.FromInt(0)
+				// expectedMaxUnavailable := intstr.FromInt(0)
```

Comment:
Line not needed? Not sure if we've got a test covering for setting in a form of an absolute number?

Reply:
Will clean up.

In regards to defaults, all tests are checking for defaults the percentage based defaults.

However, the tests also verify the `parallelism` value has been used correctly. This ensures we check for values that are absolute numbers.